README.md

```diff
@@ -372,9 +373,23 @@ Here is a summary of each parameter:
 	  had limited usage at this time and could probably be
 	  improved upon if required.
 
-       h. json_args
-       i. update_interval
-       j. history_length
+       h. json_object 
+
+          The json_object allows jschart to use JSON formatted data 
+          objects. The JSON object should be based on the following:
+
+      { 
+        json_object: {
+	      'x_axis_series': 'time',
+	      'data_series_names': [ 'time', 'data series 1', 'data series 2' ],
+	      'data': [ [ 1461027782000, 0, 1],
+                        [ 1461027783000, 4, 5] ]
+	      } 
+      }
+
+       i. json_args
+       j. update_interval
+       k. history_length
 
           These three options are only used in conjunction with the
           json_plotfile option, and most often all together (although
```

Comment:
It looks like this paragraph and the ones that follow are describing `json_plotffile`.  Should that be moved up then?

Reply:
Yes these entries should be grouped with `json_plotfile`